data/io.github.bcpierce00.unison-gui.desktop

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: MIT-0
+[Desktop Entry]
+Type=Application
+Exec=unison-gui
+Name=Unison
+GenericName=File Synchronizer
+Comment=GUI for Unison file synchronizer
+Terminal=false
+Icon=io.github.bcpierce00.unison
```

Comment:
Must the `.desktop` `Icon` entry also contain the Flatpak application ID? And does Flatpak then rename the icon files during install / Flatpak creation (e.g. `U.svg` [1] to `io.github.bcpierce00.unison.svg`)?

For the Fedora package, I rename `U.svg` to `unison.svg` and use `Icon=unison` [2,3], consistent with the Icon Theme Specification [4].

1. https://github.com/bcpierce00/unison/blob/master/icons/U.svg
2. https://docs.fedoraproject.org/en-US/packaging-guidelines/#_icon_tag_in_desktop_files
3. https://specifications.freedesktop.org/desktop-entry-spec/latest/extra-actions.html#id-1.12.4.3
4. https://specifications.freedesktop.org/icon-theme-spec/latest/#icon_lookup